spring-cloud-contract-verifier/src/main/groovy/org/springframework/cloud/contract/verifier/util/JsonToJsonPathsConverter.groovy

```diff
@@ -320,13 +320,15 @@ class JsonToJsonPathsConverter {
 	}
 
 	JsonPaths transformToJsonPathWithTestsSideValues(def json,
-			Closure parsingClosure = MapConverter.JSON_PARSING_CLOSURE) {
-		return transformToJsonPathWithValues(json, SERVER_SIDE, parsingClosure)
+			Closure parsingClosure = MapConverter.JSON_PARSING_CLOSURE,
+			boolean includeEmptyCheck = false) {
```

Comment:
Not super keen on adding a new param, but the default ability of groovy made me ok w/ it. It prevented signature ripple. The other thought was to not introduce the concept of `includeEmptyCheck` and instead have the `JsonBodyVerificationBuilder` remove the paths entry for `$.isEmpty` after calling in here. I landed here as this route was a bit cleaner and its possible we may want to do this same thing in other areas later.